train-simulation/simulator/README.md

```diff
@@ -1,15 +1,19 @@
 # Simulator
 
-This is the simulator for edgefarm. The simulation contains:
+This is the train-simulator for edgefarm. The simulation contains:
 * Temperature simulation
+* "Journey1" that replays data recorded on a regional train (location, acceleration, speed)
```

Comment:
Not yet contained.

Reply:
Changed train-simulation/simulator/docker-compose.yaml to point to the commit id containing journey1